Sources/MessagingPushAPN/RichPush/MessagingPush+NotificationServiceExtension.swift

```diff
@@ -40,6 +42,18 @@ public extension MessagingPush {
             return false
         }
 
+        // If cioHandleBadge exists and value is true
+        if let cioHandleBadge = pushContent.cioHandleBadge, cioHandleBadge  {
+            
+            let badgeCount = store.badgeCount ?? 0
+            // Update badge count in payload
+            if let newRequest = request.content.mutableCopy() as? UNMutableNotificationContent {
+                newRequest.badge = NSNumber(value: badgeCount + 1)
+                store.badgeCount = badgeCount + 1
```

Comment:
this only supports incrementing the badge, planning a follow-up to add decrementing, clearing and setting a discrete value?